pkgs/development/python-modules/dulwich.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchurl
+, pkgs, gevent, geventhttpclient, mock, fastimport }:
```

Comment:
You can explicitly pass the package to `callPackage`, e.g. `inherit (pkgs) git;`,